src/common/lru_cache.h

```diff
@@ -453,6 +461,208 @@ std::shared_ptr<CacheMetrics>
     return  cacheMetrics_;
 }
 
+// TimedLRUCache
+template <typename K,  typename V,
```

Comment:
I didn't think deeply, but `TimedLRUCache` is possibly implemented by `LRUCache`, you can package timestamp with value as a whole value for `LRUCache`.

Let's say,

```cpp
class TimedLRUCache : public LRUCachInterface<K, V> {
    struct TimestampWithValue {
         uint64_t time;
         Value value;   
    };

    LRUCache<K, TimestampWithValue> impl_;
};

```

As for `Get`, you can firstly get `TimestampWithValue` from `impl_` through the key, if you really got it, check the timestamp in `TimestampWithValue`, if it's not time out, return value in `TimestampWithValue`, otherwise, let `impl_` delete the key.

And for `Put`, the logic is almost the same, package the value with a timestamp, then put it into `impl_`.

Reply:
The main reason is if add the timeout attr in item of LRUCache will increase memory used for the scenes don't need timeout.